manifests/config.pp

```diff
@@ -4,6 +4,10 @@
 class candlepin::config {
   assert_private()
 
+  include candlepin::certificates
```

Comment:
It's better to use `contain` so that you're sure everything is done when this class is complete
```suggestion
  contain candlepin::certificates
```

Reply:
Can you explain in a bit more depth the difference between the two? I *think* I understand the flow but I am not sure.

Some context, the way this is split out right now, the certificate class needs user-group from the config class and the config class needs variables defined in the certificates class. I tried to split out a certificates class just to help with organization as the config class was getting large.

Reply:
`contain` is equal to:
```puppet
include candlepin::certificates
Class['candlepin::certificates'] -> Class['candlepin::config']
```

If you don't do so, the resources can end up being evaluated after. So you risk creating certificates after the service is started.